docs/markdown/Wrap-dependency-system-manual.md

```diff
@@ -135,7 +135,7 @@ case, the directory will be copied into `subprojects/` before applying patches.
 - `push-url` - alternative url to configure as a git push-url. Useful if
   the subproject will be developed and changes pushed upstream.
   *(since 0.37.0)*
-- `clone-recursive` - also clone submodules of the repository
+- `clone-recursive` - also clone submodules of the repository. To use it set `clone-recursive = true`.
```

Comment:
What about avoiding the repetition with:
```suggestion
- `clone-recursive=true` - also clone submodules of the repository.
```

Might as well change to `depth=X` above as well.